driver/src/main/com/mongodb/client/gridfs/GridFSDownloadStreamImpl.java

```diff
@@ -33,6 +35,8 @@
     private final long length;
     private final int chunkSizeInBytes;
     private final int numberOfChunks;
+    private MongoCursor<Document> cursor;
```

Comment:
This cursor isn't closed anywhere.  It should be closed anywhere that it's set to null (if it's not already null) and in the close method of this stream (which doesn't exist yet as there wasn't a need for it previously)